docs/devel/RELEASE.md

```diff
@@ -2,32 +2,32 @@
 
 ## Overview
 
-The release process consists of three phases: versioning, building, and
-publishing.
+The release process consists of two phases: versioning and publishing the release.
 
 Versioning involves maintaining the following files:
 
 - **CHANGELOG.md** - this file contains a list of all the important changes in each release.
 - **Makefile** - the Makefile contains a VERSION variable that defines the version of the project.
-- **docs/website/RELEASES*** - this file determines which versions of documentation are displayed
-  in the public [documentation](https://openpolicyagent.org/docs). __The first entry on the list is
-  considered to be the latest.__
 
 The steps below explain how to update these files. In addition, the repository
 should be tagged with the semantic version identifying the release.
 
-Building involves obtaining a copy of the repository, checking out the release
-tag, and building the binaries.
-
 Publishing involves creating a new *Release* on GitHub with the relevant
 CHANGELOG.md snippet and uploading the binaries from the build phase.
 
 ## Versioning
 
-1. Obtain a copy of repository.
+The steps below assume an OPA development environment has configured for the standard GitHub fork workflow. See [OPA Dev Instructions](
```

Comment:
Looks like the link didn't make it into the file.

Reply:
Fixed in latest commit